.changes/nextrelease/s3_content_type_params

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "type": "enhancement",
+        "category": "S3",
+        "description": "Set the ContentType if not already present."
```

Comment:
This was already the case, it was overriding the supplied ContentType as well. This should read:

> No longer override supplied ContentType parameter when performing a multipart upload.